languages/elixir/reference/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,124 @@
+
+## Step 2: add .docs/instructions.md
+
+This file contains instructions for the exercise. It should explicitly explain what the student needs to do (define a method with the signature `X(...)` that takes an A and returns a Z), and provide at least one example usage of that function. If there are multiple tasks within the exercise, it should provide an example of each.
+
+## Step 3: add .docs/hints.md
+
+If the student gets stuck, we will allow them to click a button requesting a hint, which shows this file. This will not be a "recommended" path and we will (softly) discourage them using it unless they can't progress without it. As such, it's worth considering that the student reading it will be a little confused/overwhelmed and maybe frustrated.
+
+The file should contain both general and task-specific "hints". These hints should be enough to unblock almost any student. They might link to the docs of the functions that need to be used.
+
+The hints should not spell out the solution, but instead point to a resource describing the solution (e.g. linking to documentation for the function to use).
+
+## Step 4: add .docs/after.md (optional)
+
+Once the student completes the exercise they will be shown this file, which should provide them with a summary of what the exercise aimed to teach. This document can also link to any additional resources that might be interesting to the student in the context of the exercise.
+
+## Step 5: add .meta/config.json
+
+This file contains metadata for the exercise. The metadata includes paths to the test file and the solution files, as well as a list of all the tests and the method they are calling.
+
+## Step 6: update languages/Elixir/config.json
+
+An entry should be added to the track's `config.json` file for the new Concept Exercise:
+
+```json
+{
+  ...
+  "exercises": {
+    "concept": [
+      ...
+      {
+        "slug": "$slug",
+        "uuid": "$uuid",
+        "concepts": ["$concept-1"],
+        "prerequisites": ["$prerequisite-1", "$prerequisite-2"]
+      }
+    ]
+  }
+}
+```
+
+## Step 7: adding track-specific files
+
+The configuration files may be copied from another exercise. But the structure of an exercise is a generic elixir mix-project.
```

Comment:
Maybe I'm reading this wrong, but can't the files be copied _because_ it is generic elixir mix project?

```suggestion
The configuration files may be copied from another exercise, as the structure of an exercise is a generic elixir mix-project.
```

Reply:
They can be, but it's probably better to make a new one with `mix new $slug_with_underscores_not_dashes` then move the contents to the `$slug` directory

Reply:
```suggestion
The configuration files may be copied from another exercise. But it would be recommended to use `mix new $slug` where `$slug` uses underscores `_` instead of dashes `-` for file generation, the move the generated structure to the `$slug` directory to match the structure in this guide.
```

Reply:
Okay good!